README.md

```diff
@@ -414,6 +419,18 @@ will ensure that all requirements in the development enviroment are pinned
 to exact versions which ensures the code will continue running as 
 expected in the future when newer versions will have been published.
 
+## 📋 Env variables
+
+You need to set the following variables for sagemaker to wor.
+
+Variable         | Required for       | Description
+---------------- | ------------------ | ----------
+PROJECTS_BUCKET  | sagemaker          | s3 bucket for data and models e.g. datalabs-data
+PROJECT_NAME     | sagemaker          | s3 prefix for specific project e.g.grants_tagger
+AWS_ACCOUNT_ID   | sagemaker          | aws account id, ask aws adminstrator
```

Comment:
Is this the actual ID (e.g. `aCampello`) or a service account ID just for SageMaker jobs? Do you also need to have `AWS_SECRET_ACCESS_KEY` and `AWS_ACCESS_KEY_ID` set up?

Reply:
this is organisational id, will clarify

you need to have the typical variables as well, will add